dashboard/config/initializers/markdown_handler.rb

```diff
@@ -6,8 +7,29 @@ module MarkdownHandler
     space_after_headers: true
   }
 
+  # Rewrite youtube iframe elements to use the fallback-player iframe instead
+  class YoutubeRewriter < Redcarpet::Render::HTML
+    def block_html(html)
+      doc = ::Nokogiri::HTML(html)
+      nodes = doc.xpath(%w(youtube youtubeeducation).map{|x| "//iframe[@src[contains(.,'#{x}.com/embed')]]"}.join(' | '))
+      nodes.each do |node|
+        if node['src']
+          regex = /(http:|https:)?\/\/(www\.)?(youtube|youtubeeducation)\.com\/embed\/([^"&?\/ ]{11})/
+          id = node['src'].match(regex)[4]
+          node['src'] = node['src'].sub(
+            regex,
+            Rails.env.development? ?
+              "/videos/embed/#{id}" :
```

Comment:
refactored this a bit (using the `url_for` helper turned out to be more difficult than it was worth, so I've simplified it in `Video#embed_url`).